src/Microsoft.VisualStudio.Editor.Razor/DefaultVisualStudioCodeDocumentProvider.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNetCore.Razor.Language;
+using Microsoft.VisualStudio.Text;
+
+namespace Microsoft.VisualStudio.Editor.Razor
+{
+    internal class DefaultVisualStudioCodeDocumentProvider : VisualStudioCodeDocumentProvider
```

Comment:
Should this have an export on it?

Reply:
Should we rename the base class to something like `TextBufferCodeDocumentProvider`